test/lib/rules/linebreak-style/unix-endings

```diff
@@ -0,0 +1,7 @@
+// This is comment containing \n and \r\n
+// Lets hope it all works!!
+
```

Comment:
When I use the above as my test case file, I get a error regarding the parsing. 
Since I have no knowledge about Solidity or its contract syntax, etc. Can you tell me what exact file should be added as the test case file?

Reply:
Never mind, used one from the other test cases.